evals/elsuite/basic/list_match.py

```diff
@@ -0,0 +1,55 @@
+from typing import Any
+
+import numpy as np
+
+import evals
+import evals.elsuite.utils
+import evals.metrics
+from evals.record import RecorderBase
+
+
+class ListMatch(evals.Eval):
```

Comment:
@logankilpatrick, the readme mentions that custom evals won't be accepted. However this eval is quite simple.

It takes an ideal list response and compares it to a provided list response, without worrying about order in the list. It returns an accuracy true if the list agrees exactly, and it also provides an f1 score that is determined by list item matching.

Would a PR that includes a ListMatch class like this be accepted at this time?